### ISSUE TITLE
Callisto update to 1.2.5 hangs VSIX installer

### ISSUE THREAD

**Issue by JPhilC** [timheuer/callisto#114]
Hi,

This morning VS 2012 told me there were updates to install. The update turns out to be Callisto 1.2.5. Unfortunately following the download the VSIX page becomes unresponsive. Non of the controls respond. I can end the Microsoft VSIX Installer (32bit) task in task manager. This allows be to close the update gallery and continue using VS 2012.

I am running VS 2012 (Version 11,0.50727.1 RTMREL) running on Windows 8 Pro (6.2.9200)

**Comment by timheuer** [comment]
If you download from the site and install does that work fine?

-th

**Comment by JPhilC** [comment]
I'll give it a try in the morning. Thks for the reply.

Sent from my Windows Phone

-----Original Message-----

**Comment by JPhilC** [comment]
Hi Tim,

I now have 1.2.5 installed. I wasn’t sure originally what you mean by “download from site” as in the past I had only used the package manager route. So unfortunately I didn’t test it as you suggested. What seemed to have happened though was that the package.config on my solution was referencing 1.2.5 but the DLL reference in the solution was still trying to point to the 1.2.4 version. I deleted the reference and the package.config (I only had Callisto installed). Following this the NuGet install worked.

I now realise that what you meant by “download from site” was download from the Visual Studio Gallery. Sorry about that.

Regards,

Phil
